tests/test_volume.py

```diff
@@ -523,7 +523,7 @@ def test_delete_clone_volume(setup_teardown_for_each_test):
             raise ex
 
 
-@pytest.mark.skipif(SKIPTEST is True,
+@pytest.mark.skipif(SKIPTEST is True or nimosclientbase.get_nimos_client().pools.list().__len__() < 2,
```

Comment:
rather than not running this testcase using skipif.. it will be better to check for the pool list length inside the code.

we already have a logic 
    ```
   # make sure we have 2 pools atleaset
        assert pool_resp.__len__() >= 2	     
```

may be u can replace assert with a if else and log the message saying .. "skipping bulk_move" as number of pools are less than 2.

Reply:
In that case the test passes, whereas technically its not even been run since it's going to fail in assert itself. I liked the skip approach better for that reason. What do you think @ar-india ?

Reply:
but in your approach too , u are not running the test case :). infact it is for this reason i have used assert and didn't wanted to go for logging way. the idea was to make sure QA has this setup. but yes sice the operation is too slow. you may want to skip 
.

if you still want to go through skip way.. then please change the log message of skip..and make it clear why you are skipping the test case..
reason="skipped this test as SKIPTEST variable is true" has to be changed.